src/discord/DiscordManager.js

```diff
@@ -41,7 +41,7 @@ class DiscordManager extends CommunicationBridge {
       channel.send({
         embed: {
           description: message,
-          color: 8311585,
+          color: 'DARK_GREEN',
```

Comment:
Since the join messages are also green it would be nice if there is a bigger difference between the chat colours, the normal chat messages could use indigo or blue instead of dark green.

Reply:
As for the colors, I was trying to stick to a dark greenish color because I was intending to use an indigo/cyan type color for the officer chat. I've made it much darker now, and you can make out a clear difference between the login messages and the normal guild messages.
![image](https://user-images.githubusercontent.com/65962094/111925307-b90bc200-8a7e-11eb-8822-85dae7d358ad.png)

Reply:
I'd still prefer if the activity messages (Join & Leave messages) doesn't share anything with normal messages so they aren't confused with each other at a glance, the chat messages could have the colour completely removed so it's just a gray colour instead, or something in the range of [#6366f1](https://www.color-hex.com/color/6366f1) to [#6495ed](https://www.color-hex.com/color/6495ed) so they're still a bright colour.